src/pages/LandingPage.js

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import { useNavigate } from 'react-router-dom';
+import useStore from '../hooks/useStore';
+import backround from '../images/background.jpg'
+
+export default function LandingPage() {
+  let navigate = useNavigate();
+  const setInstructionOneVisible = useStore(
+    state => state.setInstructionOneVisible
```

Comment:
also here, or is it because of useStore - then maybe you could do it there once?